packages/web/src/components/gcds-stepper/gcds-stepper.tsx

```diff
@@ -17,12 +17,33 @@ export class GcdsStepper {
   /**
    * Defines the current step.
    */
-  @Prop() currentStep!: number;
+  @Prop({ mutable: true }) currentStep!: number;
+  @Watch('currentStep')
+  validateCurrentStep() {
+    if (this.currentStep == undefined) {
+      this.currentStep = 1;
```

Comment:
Also, I think we shouldn't be setting this here, if we want a default value, we should set it in the declaration instead like:
```
@Prop() currentStep = 1;
@Prop() totalSteps = 3;
```

I think `validateCurrentStep()` and `validateTotalSteps()` should only be scoped to process a boolean (valid: true or false). The conditional checks are:
"Is the currentStep a positive number, and is it lesser than totalSteps?"
"Is totalSteps a positive number and is it bigger or equal to currentStep?"